src/lecturer/components/sendQuizView/SendQuizView.tsx

```diff
@@ -51,14 +52,16 @@ export function SendQuizView(props: SendQuizViewProps) {
     const studentList = props.studentList ?? [];
     const studentCount = studentList.length;
     const [students, setStudents] = useState<string[]>(selectedStudents);
-    const [time, setTime] = useState<number>(store.sendQuiz.timeInSec ?? 10);
+    const [minutes, setMinutes] = useState<number>(1);
+    const [seconds, setSeconds] = useState<number>(0);
+    const [time, setTime] = useState<number>(store.sendQuiz.timeInSec ?? 0);
```

Comment:
There is a small visual bug when you go back a step in send quiz process, or if you switch cards while being on choose time step, the inputs go back to the default event though the time is selected correctly.  
![image](https://user-images.githubusercontent.com/26445545/119273930-806a9f00-bc0d-11eb-94c7-4c450320ebb4.png)
There might be some disparity between those states and store value. Reducing it back to just time in seconds and calculating it on the go should fix it.